downstream/assemblies/platform/assembly-inventory-file-importing.adoc

```diff
@@ -0,0 +1,28 @@
+[id="assembly-inventory-file-importing"]
+
+= Inventory File Importing
+
+{ControllerNameStart} enables you to choose an inventory file from source control, rather than creating one from scratch. 
+This function is the same as for custom inventory scripts, except that the contents are obtained from source control instead of editing their contents browser. 
```

Comment:
```suggestion
This function is the same as for custom inventory scripts, except that the contents are obtained from source control instead of editing their contents in a browser. 
```
Not sure if the suggestion is correct

Reply:
Nor me. I'll go with it though.